client/my-sites/sidebar-unified/index.jsx

```diff
@@ -67,14 +67,7 @@ export const MySitesSidebarUnified = ( { path } ) => {
 					);
 				}
 
-				return (
-					<MySitesSidebarUnifiedItem
-						key={ item.slug }
-						path={ path }
```

Comment:
`path` is not used by `MySitesSidebarUnifiedItem` so we can remove and avoid it causes unnecessary re-renders.